modules/cloud-function/variables.tf

```diff
@@ -50,13 +50,13 @@ variable "environment_variables" {
 }
 
 variable "function_config" {
-  description = "Cloud function configuration."
+  description = "Cloud function configuration. Defaults to using main as entrypoint, 1 instance with 256MiB of memory, and 180 second timeout"
   type = object({
-    entry_point = string
-    instances   = number
-    memory      = number # Memory in MB
-    runtime     = string
-    timeout     = number
+    entry_point = optional(string, "main")
+    instances   = optional(number, 1)
+    memory      = optional(number, 256) # Memory in MB
```

Comment:
should we add the units to so `memory` and `timeout` become `memory_mb` `timeout_seconds`? (idea from [here](https://cloud.google.com/apis/design/naming_convention#quantities))

@ludoo wdyt?

Reply:
Added this in 2ce55e20ef87c1ffeed766a39824898b10f25e8f.